storage/buckets.rb

```diff
@@ -90,6 +90,86 @@ def create_bucket project_id:, bucket_name:
   # [END create_bucket]
 end
 
+def create_bucket_class_location project_id:, bucket_name:, location:, storage_class:
+  # [START create_bucket_class_location]
+  # project_id    = "Your Google Cloud project ID"
+  # bucket_name   = "Name of Google Cloud Storage bucket to create"
+  # location      = "Location of your Google Cloud Storage bucket"
+  # storage_class = "Storage class of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
+  bucket  = storage.create_bucket bucket_name,
+                                  location:      location,
+                                  storage_class: storage_class
+
+  puts "Created bucket #{bucket.name} in #{location} with storage class #{storage_class}"
+  # [END create_bucket_class_location]
+end
+
+def get_bucket_labels project_id:, bucket_name:
```

Comment:
`get_bucket_labels` --> `list_bucket_labels`

For CRUD types of samples, we name them `get_single_resource` or `list_resources` like `get_bucket` and `list_buckets`

If this conflicts with our defined standard doc region tags, then this is OK. I just don't like it 😝